app/src/components/HomePage/HomePage.jsx

```diff
@@ -1,19 +1,17 @@
+"use client";
```

Comment:
You don't need to use `'use client'` here since you're using Vite instead of Next.js